src/graphnode.ts

```diff
@@ -43,7 +43,8 @@ export function createApolloClient(options: {
 
   const wsLink = new WebSocketLink({
     options: {
-      reconnect: true
+      lazy: true,
```

Comment:
I think lazy should be set to false https://github.com/apollographql/subscriptions-transport-ws/blob/2b0eefa438409263d0e38e3a39d062e3def41218/README.md#subscriptionclient